build/lib/uposham/turning_point_coord_difference.py

```diff
@@ -0,0 +1,393 @@
+# -*- coding: utf-8 -*-
```

Comment:
Looks like `build/` snuck back into the repo - that directory should not be checked in.

Reply:
I think the housekeeping branch under PR #57 has taken care of this. I'll check this again after merge.